src/webhook/mutation/pod_mutator.go

```diff
@@ -465,7 +472,8 @@ func (m *podMutator) getBasicData(pod *corev1.Pod) (
 	failurePolicy string,
 	image string,
 ) {
-	flavor = kubeobjects.GetField(pod.Annotations, dtwebhook.AnnotationFlavor, arch.FlavorMultidistro)
+	// arch.Flavor defaults to the platform-specific value
```

Comment:
```suggestion

	// arch.Flavor defaults to the platform-specific value
```

Reply:
Missed this in the first commit, as the msg was hidden.

Fixed in latest commit.